SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -1,31 +1,20 @@
-<%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
+# SpatialOS GDK for Unreal toolbar
 
-# SpatialOS Unreal GDK toolbar
+The GDK toolbar provides several functions required for building and launching your client- and server-workers from inside the Unreal Editor.
 
-The SpatialOS Unreal GDK toolbar is an easy way to run and alter [`spatial` command-line tool](https://docs.improbable.io/reference/latest/shared/glossary#the-spatial-command-line-tool) (SpatialOS documentation) commands from inside the Unreal Editor.
+## Terms used in this doc
+`<ProjectRoot>` - The root folder of your Unreal project.  
+`<GameRoot>` - The folder containing your game's `.uproject` and source folder (for example, `<ProjectRoot>/ShooterGame/`).  
 
-## Add the SpatialOS Unreal GDK toolbar to your Unreal project
+## Add the GDK toolbar to your Unreal project
```

Comment:
Let's add a note that this is already done for you if you based your project off of StarterProject